src/theme/index.tsx

```diff
@@ -42,44 +42,47 @@ export function colors(darkMode: boolean): Colors {
 
     // text
     text1: darkMode ? '#FFFFFF' : '#000000',
-    text2: darkMode ? '#C3C5CB' : '#565A69',
+    text2: darkMode ? '#DCDCDC' : '#565A69',
```

Comment:
We shouldn't modify any original file. I can help to make a "Mod" of this

Reply:
Yes, understand. There are multiple mods in this file just FYI.

Reply:
I propose we create `custom/theme/index.tsx` with

```ts
import {colors as originColors} from '@/theme' // after #11 is merged

export function colors(darkMode: boolean): Colors {
  return {
    ...originColors(darkMode),
   <your_overrides>
}
```